orchagent/aclorch.cpp

```diff
@@ -3049,11 +3253,78 @@ bool AclOrch::updateAclRule(string table_id, string rule_id, string attr_name, v
     return true;
 }
 
+bool AclOrch::updateAclRule(string table_id, string rule_id, bool enableCounter)
```

Comment:
I think this interface is not covered by unit test. Could you please check?

Reply:
@bingwang-ms sure. I will add a UT.

Reply:
@bingwang-ms done